src/presenter/commander.ts

```diff
@@ -54,26 +54,48 @@ function LaunchPresentation() {
   const currentDisplay = screen.getDisplayNearestPoint(
     screen.getCursorScreenPoint()
   );
-  const otherDisplay = _.find(
+
+  const devMode = process.env.NODE_ENV === 'development';
+
+  let otherDisplay = _.find(
     displays,
     (d: Display) => d.id != currentDisplay.id
   );
 
+  if (!otherDisplay && devMode) {
+    otherDisplay = displays[0];
+  }
+
   if (otherDisplay) {
-    let presentationWindow: Electron.BrowserWindow = new remote.BrowserWindow({
+    const windowOptions: any = {
       frame: false,
       autoHideMenuBar: true,
       fullscreen: true,
       x: otherDisplay.bounds.x + 50,
       y: otherDisplay.bounds.y + 50,
       parent: remote.getCurrentWindow(),
       backgroundColor: '#000000',
-    });
+    };
+
+    // check for development mode
+    if (process.env.NODE_ENV === 'development') {
+      windowOptions.fullscreen = false;
+      windowOptions.x = 50;
+      windowOptions.y = 50;
+      windowOptions.width = 1920 / 2;
+      windowOptions.height = 1080 / 2;
+      windowOptions.frame = true;
```

Comment:
You can turn off the frame... but it's annoying as you can't move the window around easily